components/screens/DepositScreen.tsx

```diff
@@ -20,8 +24,49 @@ export const TemplateScreen = ({ title, body }) => {
                 <HeaderText style={{ textAlign: "center" }}>
                     {title}
                 </HeaderText>
+                <View style={styles.container}>
+                    <View style={styles.backgroundV2}>
+                        <StandardText>
+                           V2 Markets
+                        </StandardText>
+                    </View>
+                    <View width={200}/>
+                    <View style={styles.depositA}>
+                        <View style={styles.containerDeposit}>
+                            <DepositText>Deposit APY</DepositText>
+                        </View>
+                        <View style={styles.containerAPY}>
+                            <TokenIcon address={aaveAddress} size={13}/>
+                            <SecondaryText style={{ marginLeft: 5 }}>
+                                Reward Aave
+                            </SecondaryText>
+                        </View>
+                    </View>
+                </View>
             </View>
             { body }
         </View>
     );
 };
+
+const styles = StyleSheet.create({
+    container: {
+        flexDirection: "row",
+        paddingTop: 20,
+        alignItems: "center",
+    },
+    containerAPY: {
+        flexDirection: "row",
+    },
+    backgroundV2: {
+        backgroundColor: STYLES.backgrounds.secondaryBackground,
+        padding: 7,
+        borderRadius: 8,
+    },
+    containerDeposit: {
+        padding: 8,
+    },
```

Comment:
Deposit APY isn't align correctly -- the reason is because you added padding on the right side of it!, let's change this `padding: 8` to `marginBottom: 8` and take a look at how it looks like ;)